packages/vaadin-lumo-styles/src/components/card.css

```diff
@@ -4,240 +4,9 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 @media lumo_components_card {
-  :host {
-    display: flex;
-    flex-direction: column;
-    box-sizing: border-box;
-    padding: var(--_padding);
-    gap: var(--_gap);
-    --_padding: var(--vaadin-card-padding, 1em);
-    --_gap: var(--vaadin-card-gap, 1em);
-    --_media: 0;
-    --_title: 0;
-    --_subtitle: 0;
-    --_header: max(var(--_header-prefix), var(--_title), var(--_subtitle), var(--_header-suffix));
-    --_header-prefix: 0;
-    --_header-suffix: 0;
-    --_content: 0;
-    --_footer: 0;
-    background: var(--vaadin-card-background);
-    border-radius: var(--vaadin-card-border-radius);
-    box-shadow: var(--vaadin-card-shadow);
-    position: relative;
-  }
-
-  :host([hidden]) {
-    display: none !important;
-  }
-
-  :host(:not([theme~='horizontal'])) {
-    justify-content: space-between;
-  }
-
-  :host([_m]) {
-    --_media: 1;
-  }
-
-  :host([_t]) {
-    --_title: 1;
-  }
-
-  :host([_st]) {
-    --_subtitle: 1;
-  }
-
-  :host([_h]) {
-    --_header: 1;
-    --_title: 0;
-    --_subtitle: 0;
-  }
-
-  :host([_hp]) {
-    --_header-prefix: 1;
-  }
-
-  :host([_hs]) {
-    --_header-suffix: 1;
-  }
-
-  :host([_c]) {
-    --_content: 1;
-  }
-
-  :host([_f]) {
-    --_footer: 1;
-  }
-
-  [part='media'],
-  [part='header'],
-  [part='content'],
-  [part='footer'] {
-    display: none;
-  }
-
-  :host([_m]) [part='media'],
-  :host([_c]) [part='content'] {
-    display: block;
-  }
-
-  :host([_f]) [part='footer'] {
-    display: flex;
-    gap: var(--_gap);
-  }
-
-  :host(:is([_h], [_t], [_st], [_hp], [_hs])) [part='header'] {
-    display: grid;
-    align-items: center;
-    gap: 0 var(--_gap);
```

Comment:
The gap for the header is not correctly set, which causes some screenshots like `title-subtitle` to fail.
The JS version has `--_gap` set to `1rem` but base styles use `--vaadin-gap-container-inline`.